layers/core_checks/cc_render_pass.cpp

```diff
@@ -3069,6 +3069,63 @@ bool CoreChecks::ValidateRenderingAttachmentInfo(VkCommandBuffer commandBuffer,
         }
     }
 
+    if (const auto attachment_flags = vku::FindStructInPNextChain<VkRenderingAttachmentFlagsInfoKHR>(attachment_info.pNext)) {
+        if (attachment_flags->flags & (VK_RENDERING_ATTACHMENT_RESOLVE_SKIP_TRANSFER_FUNCTION_BIT_KHR |
+                                       VK_RENDERING_ATTACHMENT_RESOLVE_ENABLE_TRANSFER_FUNCTION_BIT_KHR)) {
+            if (!vkuFormatIsSRGB(image_view_format)) {
+                const LogObjectList objlist(commandBuffer, attachment_info.imageView);
+                skip |= LogError("VUID-VkRenderingAttachmentInfo-pNext-11752", objlist,
+                                 attachment_loc.pNext(Struct::VkRenderingAttachmentFlagsInfoKHR, Field::flags),
+                                 "is %s but image view has format %s.",
+                                 string_VkRenderingAttachmentFlagsKHR(attachment_flags->flags).c_str(),
+                                 string_VkFormat(image_view_format));
+            }
+            if (attachment_info.resolveMode != VK_RESOLVE_MODE_AVERAGE_BIT) {
+                const LogObjectList objlist(commandBuffer, attachment_info.imageView);
+                skip |= LogError(
+                    "VUID-VkRenderingAttachmentInfo-pNext-11753", objlist, attachment_loc.dot(Field::resolveMode),
+                    "is %s but a VkRenderingAttachmentFlagsInfoKHR is included in the attachment pNext chain with flags (%s).",
+                    string_VkResolveModeFlags(attachment_info.resolveMode).c_str(),
+                    string_VkRenderingAttachmentFlagsKHR(attachment_flags->flags).c_str());
+            }
+        }
+        if (attachment_flags->flags & VK_RENDERING_ATTACHMENT_INPUT_ATTACHMENT_FEEDBACK_BIT_KHR) {
+            if (!(image_view_state->image_state->create_info.usage & VK_IMAGE_USAGE_INPUT_ATTACHMENT_BIT)) {
+                const LogObjectList objlist(commandBuffer, attachment_info.imageView);
+                skip |=
+                    LogError("VUID-VkRenderingAttachmentInfo-pNext-11754", objlist,
+                             attachment_loc.pNext(Struct::VkRenderingAttachmentFlagsInfoKHR, Field::flags),
+                             "is %s but image has usage %s.", string_VkRenderingAttachmentFlagsKHR(attachment_flags->flags).c_str(),
+                             string_VkImageUsageFlags(image_view_state->image_state->create_info.usage).c_str());
+            }
+            if (!enabled_features.dynamicRenderingLocalRead) {
+                const LogObjectList objlist(commandBuffer);
+                skip |= LogError("VUID-VkRenderingAttachmentFlagsInfoKHR-flags-11755", objlist,
```

Comment:
```suggestion
                skip |= LogError("VUID-VkRenderingAttachmentFlagsInfoKHR-flags-11755", commandBuffer,
```

same with those below